src/test/java/no/nav/arxaas/GenerateEdgeCaseData.java

```diff
@@ -369,11 +368,104 @@ public static MockMultipartFile testMetaDataWrongFormat() {
         return new MockMultipartFile("metadata", "","application/json", testFormData_metadata_2quasi_wrong_format().getBytes());
     }
 
+    public static MockMultipartFile testMetaData3Quasi() {
+        return new MockMultipartFile("metadata", "","application/json", testFormData_metadata_3quasi().getBytes());
+    }
+
+    public static MockMultipartFile testMetaDataMissingPrivacyModel() {
+        return new MockMultipartFile("metadata", "","application/json", testFormData_metadata_missing_privacy_models().getBytes());
+    }
+
+    public static MockMultipartFile testMetaDataToMuchPrivacyModel() {
+        return new MockMultipartFile("metadata", "","application/json", testFormData_metadata_privacy_models_for_non_existing_attribute().getBytes());
+    }
+
+    public static MockMultipartFile testMetaDataWrongPrivacyFormat(){
+        return new  MockMultipartFile("metadata", "","application/json", testFormData_metadata_wrong_privacy_model_format().getBytes());
+    }
+
+    public static MockMultipartFile testMetaDataSuppressionHigherThan1(){
+        return new  MockMultipartFile("metadata", "","application/json", testFormData_metadata_2quasi_supression_higher_than_1().getBytes());
+    }
+
+    public static MockMultipartFile testMetaDataSuppressionLowerThan0(){
+        return new  MockMultipartFile("metadata", "","application/json", testFormData_metadata_2quasi_supression_lower_than_0().getBytes());
+    }
+
+    public static MockMultipartFile testMetaData1Sensitive(){
+        return new  MockMultipartFile("metadata", "","application/json", testFormData_metadata_1_sensitive().getBytes());
+    }
+
+    public static MockMultipartFile testZipcodeHierarchyWrongFormat() {
+        return (MockMultipartFile) GenerateTestData.makeMockMultipartFile("./src/test/resources/testZipcodeHierarchyComma.csv","hierarchies","text/csv");
+    }
+
+    public static MockMultipartFile testZipcodeHierarchyMoreData() {
+        return (MockMultipartFile) GenerateTestData.makeMockMultipartFile("./src/test/resources/testZipcodeHierarchyMore.csv","hierarchies","text/csv");
+    }
+
     private static String testFormData_metadata_2quasi_wrong_format(){
         return "{\"attributes\":[{\"field\":\"aGe\",\"attributeTypeModel\":\"IDENTIFYING\",\"hierarchy\":null}," +
                 "{\"field\":\"GendEr\",\"attributeTypeModel\":\"QUASIIDENTIFYING\",\"hierarchy\":null}," +
                 "{\"field\":\"ZiPcoDe\",\"attributeTypeModel\":\"QUASIIDENTIFYING\",\"hierarchy\":1}]," +
                 "\"privacyModels\":[{\"privacyModel\":\"KANONYMITY\",\"params\":{\"k\":5}}]," +
                 "\"suppressionLimit\":0.02}";
     }
-}
+
+    private static String testFormData_metadata_3quasi(){
```

Comment:
Probably better with words not numbers i.e 3 -> three

Reply:
solved